lib/model/unreads.dart

```diff
@@ -70,18 +80,27 @@ class Unreads extends ChangeNotifier {
   }
 
   Unreads._({
+    required totalCount,
     required this.streams,
     required this.dms,
     required this.mentions,
     required this.oldUnreadsMissing,
     required this.selfUserId,
-  });
+  }) : _totalCount = totalCount;
 
-  // TODO excluded for now; would need to handle nuances around muting etc.
-  // int count;
+  /// Total unread messages.
```

Comment:
Hmm, but do we have a use case for this?

I don't immediately see one — I feel like any place that we might want a count like this one, we'd actually want the version that accounts for muting.

If we don't have a use case where it'd be valid to use, then given that it looks appealingly *like* something one might want in situations where in fact it'd be buggy to use it, it's probably best not to have it around.

(If we did have reason to keep it around, I'd want the doc to clearly warn that it's not the "total number of unreads" one wants in the UI.)